UnbxdSearch_IO.js

```diff
@@ -1868,8 +1868,8 @@ var unbxdSearchInit = function (jQuery, Handlebars) {
 
             // When there are no facets returned in the search response
             if (!obj.facets) {
-                jQuery(this.options.facetContainerSelector).style.display = "none";
-                jQuery(this.options.searchResultContainer).style.width = "100%";
+                jQuery(this.options.facetContainerWrapperSelector)[0].style.display = "none";
+                jQuery(this.options.searchResultContainer)[0].style.width = "100%";
```

Comment:
here we are assuming they will always give class name, can we instead just check if its an array do the [0] or else do not do it (for example if "id" is given)

Reply:
<img width="1274" alt="Screenshot 2020-08-11 at 10 45 01 PM" src="https://user-images.githubusercontent.com/47173038/89927719-5dde3b00-dc24-11ea-93af-242005d5eaab.png">